.dir-locals.el

```diff
@@ -219,6 +177,7 @@
                             (t
                              normal-indent)))))))
 
-            (advice-add #'calculate-lisp-indent :around #'wrap~calculate-lisp-indent)))
+            (advice-add #'calculate-lisp-indent :around #'wrap-calculate-lisp-indent)))
```

Comment:
nit: I think we can check Emacs version here and don't need to advice it if the version is >= "28.0"